src/app/models/partner-info.ts

```diff
@@ -0,0 +1,26 @@
+import {useState, useLayoutEffect} from 'react';
```

Comment:
This was previously reviews.js. We aren't using reviews anymore, so it's really just some partner info.